internal/byok/command_create.go

```diff
@@ -43,15 +41,62 @@ var encryptionKeyPolicyAws = template.Must(template.New("encryptionKeyPolicyAws"
 }`))
 
 const (
+	failedToRenderKeyPolicyErrorMsg = "BYOK error: failed to render key policy"
+
 	keyVaultCryptoServiceEncryptionUser = "e147488a-f6f5-4113-8e2d-b22465e65bf6"
 	keyVaultReader                      = "21090545-7ca7-4776-b22c-e363652d74d2"
+
+	defaultGcpRoleName = "custom_kms_role"
 )
 
+type gcpPolicyMetadata struct {
+	project  string
+	location string
+	keyRing  string
+	key      string
+	group    string
+}
+
+func (g gcpPolicyMetadata) renderPolicy() string {
+	return fmt.Sprintf(`
+gcloud iam roles create %[1]s \
+	--project=%[2]s \
+	--description="Grant necessary permissions for Confluent to access KMS key" \
+	--permissions=cloudkms.cryptoKeyVersions.useToDecrypt,cloudkms.cryptoKeyVersions.useToEncrypt,cloudkms.cryptoKeys.get  && \
+gcloud kms keys add-iam-policy-binding %[3]s \
+	--project=%[2]s \
+	--keyring='%[4]s' \
```

Comment:
We normallyuse " instead of ' in suggested commands. Can we use double quotes here or could there be special characters in these values?